shaky/src/main/res/values/shaky_attrs.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright (C) 2016 LinkedIn Corp.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<resources>
+    <!-- Common -->
+    <attr name="shakyBackgroundColor" format="reference|color" />
+    <attr name="shakyTitleColor" format="reference|color" />
+    <attr name="shakyContentColor" format="reference|color" />
+    <attr name="shakyToolbarColor" format="reference|color" />
+    <attr name="shakyToolbarTitleTextColor" format="reference|color" />
+    <attr name="shakyToolbarControlColor" format="reference|color" />
+    <attr name="shakyToolbarSize" format="reference|dimension" />
+
+    <!-- Category Select -->
+    <attr name="shakySelectBackgroundColor" format="reference|color" />
+    <attr name="shakyCategoryListItemTitleColor" format="reference|color" />
+    <attr name="shakyCategoryListItemSubtitleColor" format="reference|color" />
+
+    <!-- Image Editor -->
+    <attr name="shakyImageEditorBackgroundColor" format="reference|color" />
+    <attr name="shakyImageEditorButtonColor" format="reference|color" />
+    <attr name="shakyImageEditorButtonTextColor" format="reference|color" />
```

Comment:
For all buttons, we should be setting an overall button style instead of just the text and background colors. This allows for greater flexibility with styling buttons, like adding a start/end drawable for example.